src/app/components/profile-location/profile-location.component.css

```diff
@@ -3,11 +3,28 @@
 }
 
 h4{
-   
+
     font-weight:bold;
     text-align:left;
 }
 
+h5{
+
+    font-weight:bold;
+    text-align:left;
+}
+
+
+/* error */
+.error {
+    margin-left:-8px;
+    width:320px !important;
+    padding-left:0px !important;
```

Comment:
why are you using !important. This should be avoided almost always. Have you tried adding greater specificity? Just making it "html .error" will make this command more important and making it "html body .error" even more so.

Reply:
(of course using html elements that are inherently related to where your style is used is a better practice than using top level tags)

Reply:
made attempts to increase specificity but it just will not cooperate. will have to leave them in for now